document llm-as-judge eval additional variable substitution (#34029)
DataDog/documentation: content/en/llm_observability/evaluations/custom_llm_as_a_judge_evaluations.md

```
{{% /collapse-content %}}

> 1. In the **User** field, provide your user prompt. Explicitly specify what parts of the span to evaluate. Usually this will be Span Input (`{{span_input}}`) and/or Span Output (`{{span_output}}`). There are other variables available, which you can see when you type `{{`.

Author: You're right. Changed it to `8.`.

### Test and preview

> The pane on the right will show **Filtered Spans** (or traces) corresponding to the evaluation scope you configured.

Reviewer: Maybe rephrase to `the panel on the right shows **Filtered Spans** (or traces) corresponding to the configured evaluation scope.`

> Selecting a span shows JSON data available for use in an evaluation. The **Test Evaluation** button tests the evaluation against this data. When this button is clicked, inputs to your evaluation are pre-filled with data from the span. Click **Run** to test.

Reviewer 2: ```suggestion
Select a span to show JSON data available for use in an evaluation. Then, click **Test Evaluation** to pre-fill inputs to your evaluation with data from the span, and click **Run** to test.
```
Does this keep the meaning of this section?

> Clicking one of these spans will show JSON data available for use in your evaluation, with a **Test Evaluation** button to test your evaluation against the data. When you click this button, inputs to your evaluation will be pre-filled with data from the span. Click **Run** to test.

Reviewer: Maybe rephrase to `Selecting a span shows the JSON data available for use in an evaluation. The **Test Evaluation** button tests the evaluation against this data. When this button is clicked, inputs to the evaluation are pre-filled with data from the span. Click **Run** to test.``

> From the JSON, you can also use the three-dots menu to the left of each row. Select **Add variable to message** to add that data as a variable in your user message.

Reviewer 2: ```suggestion
To add a JSON data as a variable in your user message, use the three-dots menu to the left of each JSON row JSON  and select **Add variable to message**. 
```

> From the JSON, you can also use the three-dots menu to the left of each row and select **Add variable to message** to add that row's data as a variable in your user message.

Reviewer 3: > to add that row's data as a variable in your user message
nit: is it' the row's data or the section's data? i think users can select a section which is larger than a row. so maybe "to add that section's data as a variable in your user message"?

> {{< img src="llm_observability/evaluations/custom_llm_judge_2-4.png" alt="The menu contents of the JSON view in the custom evaluation configuration right pane, displaying the Add variable to message option." >}}

Reviewer 3: this image is a bit blurry, can you pls make a higher res one
